build/pxc-entrypoint.sh

```diff
@@ -197,6 +197,29 @@ if [[ $MYSQL_VERSION == '8.0' ]]; then
 	fi
 fi
 
+auth_plugin=${DEFAULT_AUTHENTICATION_PLUGIN}
+if [[ -f /var/lib/mysql/auth_plugin ]]; then
+	prev_auth_plugin=$(cat /var/lib/mysql/auth_plugin)
+	if [[ ${prev_auth_plugin} != "mysql_native_password" && ${auth_plugin} == "mysql_native_password" ]]; then
+		echo "FATAL: It's forbidden to switch from ${prev_auth_plugin} to ${auth_plugin}."
+		echo "You can remove /var/lib/mysql/auth_plugin to force the switch."
+		exit 1
+	fi
+fi
+
+if [[ -z ${auth_plugin} ]]; then
+	auth_plugin="mysql_native_password"
+elif [[ $MYSQL_VERSION == '5.7' ]]; then
+	auth_plugin="mysql_native_password"
+fi
+
+sed -i "/default_authentication_plugin/d" $CFG
+if [[ $MYSQL_VERSION == '8.0' && $MYSQL_PATCH_VERSION -ge 27 ]]; then
+	sed -i "/\[mysqld\]/a authentication_policy=${auth_plugin},," $CFG
```

Comment:
Why do we need to have` ,,`?

Reply:
https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_authentication_policy